<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCCTVConfiguration</class>
 <widget class="QWidget" name="frmCCTVConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1296</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QGroupBox" name="gbxHeader">
         <property name="title">
          <string>Header</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="lblCCTV">
                <property name="text">
                 <string>CCTV</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblVersion">
                <property name="text">
                 <string>Version</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblDate">
                <property name="text">
                 <string>Date</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblIntersectionId">
                <property name="text">
                 <string>Intersection ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblDeviceIp">
                <property name="text">
                 <string>Device IP</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblCameraPosition">
                <property name="text">
                 <string>Camera Position</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblCameraPositionLat">
                <property name="text">
                 <string>Lat.</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblCameraPositionLng">
                <property name="text">
                 <string>Lng.</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLineEdit" name="txtCCTV"/>
              </item>
              <item>
               <widget class="QLineEdit" name="txtVersion"/>
              </item>
              <item>
               <widget class="QLineEdit" name="txtDate"/>
              </item>
              <item>
               <widget class="QLineEdit" name="txtIntersectionId"/>
              </item>
              <item>
               <widget class="QLineEdit" name="txtDeviceIp"/>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="txtCameraPositionLat"/>
              </item>
              <item>
               <widget class="QLineEdit" name="txtCameraPositionLng"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="gbxRoadInformation">
         <property name="title">
          <string>RoadInformation</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="lblReferencePoints">
              <property name="text">
               <string>Reference Points</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QTableWidget" name="tblReferencePoints"/>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="btnDeletePoint">
                  <property name="text">
                   <string>Delete Point</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnAddPoint">
                  <property name="text">
                   <string>Add Point</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="gbxVirtualGate">
        <property name="title">
         <string>Virtual Gate</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QWidget" name="">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="lblRoad">
               <property name="text">
                <string>Road</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QTableWidget" name="tblRoad"/>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QPushButton" name="btnDeleteRoad">
                   <property name="text">
                    <string>Delete Road</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btnAddRoad">
                   <property name="text">
                    <string>Add Road</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="lblLane">
               <property name="text">
                <string>Lane</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QTableWidget" name="tblLane"/>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QPushButton" name="btnDeleteLane">
                   <property name="text">
                    <string>Delete Lane</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btnAddLane">
                   <property name="text">
                    <string>Add Lane</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
